client/src/components/Overview/ImageGallery.jsx

```diff
@@ -42,14 +43,27 @@ const StyledRightButton = styled.section`
   position:absolute;
   right:5%;
   top:40%;
+  width: 40px;
+  height: 40px;
+  align-items: center;
+  transform: translateY(-50%);
   font-size:x-large;
+  background: rgba(255, 255, 255, .8);
+  border-radius:50%;
 `;
 
+
 const StyledLeftButton = styled.section`
   position:absolute;
   left:120px;
   top:40%;
   font-size:x-large;
+  width: 40px;
+  height: 40px;
+  align-items: center;
+  transform: translateY(-50%);
+  background: rgba(255, 255, 255, .8);
+  border-radius:50%;
```

Comment:
We can probably combine these attributes into a basic button styled component since we're using this for at least 2 buttons each